.github/workflows/checks.yaml

```diff
@@ -37,4 +40,11 @@ jobs:
         godot --path . --headless --import
     - name: Run tests
       run: |
-        godot --path . --headless --script addons/gut/gut_cmdln.gd -gexit
+        godot --path . --headless --script addons/gut/gut_cmdln.gd -gjunit_xml_file=test-report.xml -gexit
+    - name: Publish test results
+      uses: EnricoMi/publish-unit-test-result-action@v2
+      if: always()
+      with:
+        files: test-report.xml
+        comment_mode: failures
+        test_file_prefix: "-res://"
```

Comment:
This doesn't seem to work; so the annotations don't get lined up with the source files.